toonz/sources/toonz/xsheetdragtool.cpp

```diff
@@ -1608,9 +1608,10 @@ class ColumnMoveDragTool final : public XsheetGUI::DragTool {
     assert(m_lastCol == *indices.begin());
 
     if (col < 0) col = 0;
-    if (col == m_lastCol) return;
-    int dCol  = col - m_lastCol;
+    if (col == (m_lastCol - m_offset)) return;
+    int dCol  = col - (m_lastCol - m_offset);
     m_lastCol = col;
+    m_offset  = 0;
```

Comment:
@manongjohn 
I detected a crash problem on the column drag.
To reproduce:

1. Put some level on the Column 3
1. Select the Column 1,2 and 3
1. Grab the Column 3 and drag the selection to left (on the xsheet mode, Up on the timeline mode).

How about modifying here like as follows to fix the problem?:
```
-------------------- toonz/sources/toonz/xsheetdragtool.cpp --------------------
index 2902bed..c750c29 100644
@@ -1609,9 +1609,9 @@ public:
 
     if (col < 0) col = 0;
     if (col == (m_lastCol - m_offset)) return;
     int dCol  = col - (m_lastCol - m_offset);
-    m_lastCol = col;
-    m_offset  = 0;
+    if (dCol < -(*indices.begin())) return;
+    m_lastCol = col + m_offset;
 
     assert(*indices.begin() + dCol >= 0);
```

Other modifications seems to be perfect. Thanks!

Reply:
Thanks for reviewing. Actually. I think I will revert that specific change and take it out of this PR.  When I was working on the timeline flip PR, I had to include the fix there and did run into this issue.  I corrected it in that PR as it needed a bit more work so this change would be pointless.

Reply:
OK, it is fine to move this fix to the timeline flip PR. I'm planning to review that by the end of this week.